pymerc/game/recipe.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional, List
+from pymerc.api.models.common import Item, InventoryAccountAsset, InventoryManager
+from pymerc.api.models.static import Recipe as RecipeModel
+
+if TYPE_CHECKING:
+    from pymerc.client import Client
+    from pymerc.game.building import Building
+
+
+class Recipe:
+    def __init__(self, client: Client, recipe_name: str, recipe: RecipeModel | None = None):
+        self._client = client
+        self.recipe_name = recipe_name
+        self.labor: Optional[float] = 0.0
+        self.data: Optional[RecipeModel] = recipe
+        if recipe is not None:
+            self.data = recipe
+
+    async def load(self):
+        """Loads the data for the recipe and calculates labor required."""
+        if self.data is None:
+            recipes = await self._client.static_api.get_recipes()
+            for recipe in recipes:
+                if recipe.name.value == self.recipe_name:
+                    self.data = recipe
+        if self.data is None:
+            raise ValueError(f"Recipe {self.recipe_name} not found")
+        else:
+            self.labor = self._determine_labor_required()
+
+    def _determine_labor_required(self) -> float:
+        """Calculates the labor required for the recipe."""
+        for input_ingredient in self.data.inputs:
+            if input_ingredient.product == Item.Labour:
+                return input_ingredient.amount
+        return 0.0
+
+    def get_labor(self) -> Optional[float]:
+        """Returns the labor required for the recipe."""
+        return self.labor
+
+    def calculate_target_labor(
+            self, target: float, inventory_assets: Optional[dict[Item, InventoryAccountAsset]] = None,
+            inventory_managers: Optional[List[InventoryManager]] = None
+    ) -> float:
+        """Calculates the labor required for the given target multiplier.
+
+        Args:
+            target (float): The target percentage multiplier for the recipe.
+            inventory_assets (Optional[List[InventoryAccountAsset]]): The list of inventory assets.
+            inventory_managers (Optional[List[InventoryManager]]): The list of inventory managers.
+
+        Returns:
+            float: The labor required for the target multiplier.
+        """
+        inventory_assets = inventory_assets or {}
```

Comment:
You can just set the default value to `{}` in the function arguments. Same for the line below this one.